crates/bevy_ecs/src/system/query.rs

```diff
@@ -547,7 +547,7 @@ where
     /// # use bevy_ecs::prelude::*;
     /// #
     /// # struct Person { name: String, credits: u32 }
-    /// # struct Target { entity: Entity }
+    /// # struct Winner { entity: Entity }
```

Comment:
There's a couple places (at least one below and above this) where `Target` was used as well <3

Reply:
Yes, I saw. I did those changes, but at the end I opted for a less convoluted example...